internal/controlplane/handlers_oauth.go

```diff
@@ -291,6 +291,19 @@ func (s *Server) processOAuthCallback(ctx context.Context, w http.ResponseWriter
 		return fmt.Errorf("error encoding token: %w", err)
 	}
 
+	// Verify if the token user is a member of the ownerFilter (Organization)
+	if stateData.OwnerFilter.Valid {
+		member, err := s.validateOwnerFilter(ctx, token, stateData.OwnerFilter.String)
+		if (err != nil) {
+			return err
+		}
+		if !member {
+			return newHttpError(http.StatusForbidden, "Invalid OwnerFilter Provided").SetContents(
+				"You do not have access to the organization %s, specified in the owner filter."+
+					" Please ensure that you are a member of the GitHub organization and try again.", stateData.OwnerFilter.String)
+		}
+	}
```

Comment:
We don't need to return the `member` boolean from the helper, we can simply return the error, and then this second `if` statement is not needed.

The helper would be:
```
func (s *Server) validateOwnerFilter(ctx context.Context, token *oauth2.Token, ownerFilter string) error {
	member, err := s.ghProviders.ValidateOrgMembershipForToken(ctx, token, ownerFilter)
	if err != nil {
		return fmt.Errorf("error checking organization membership: %w", err)
	}

	if !member {
		return newHttpError(http.StatusForbidden, "Invalid OwnerFilter Provided").SetContents(
			"You do not have access to the organization %s, specified in the owner filter."+
				" Please ensure that you are a member of the GitHub organization and try again.", ownerFilter)
	}
	return nil
}
```

Reply:
I recall running `make lint` before i sent the change to upstream, but i might have missed running it after the last refactor. Thanks for catching this.